setup.py

```diff
@@ -47,6 +47,7 @@ def long_desc(root_path):
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.1',
         'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4.0',
     ],
     zip_safe=False,
     tests_require=['Django>=2.2'],
```

Comment:
Shouldn't `install_requires` also say `>=2.2` ? I don't think there's even a need to put `django` in `tests_require` if it's in `install_requires`.

Reply:
Good point. I update the `install_requires` and removed `tests_require`